roles/invoiceninja/defaults/main.yml

```diff
@@ -0,0 +1,157 @@
+##########################################################################
+# Title:            Community: Invoice Ninja | Default Variables         #
+# Author(s):        RXWatcher, kowalski                                  #
+# URL:              https://github.com/saltyorg/Community                #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+invoiceninja_name: invoiceninja
+
+################################
+# Paths
+################################
+
+invoiceninja_paths_folder: "{{ invoiceninja_name }}"
+invoiceninja_paths_location: "{{ server_appdata_path }}/{{ invoiceninja_paths_folder }}"
+invoiceninja_paths_folders_list:
+  - "{{ invoiceninja_paths_location }}"
+  - "{{ invoiceninja_paths_location }}/public/logo"
+  - "{{ invoiceninja_paths_location }}/storage"
+  - "{{ invoiceninja_paths_location }}/logs"
+  - "{{ invoiceninja_paths_location }}/logs/cron"
+  - "{{ invoiceninja_paths_location }}/logs/nginx"
+invoiceninja_paths_config_location: "{{ invoiceninja_paths_location }}/env"
+
+################################
+# Web
+################################
+
+invoiceninja_web_subdomain: "{{ invoiceninja.subdomain|default('invoiceninja',true) }}"
```

Comment:
We're going away from this type of stuff. Overwrites will only happen in inventory.